src/components/index.js

```diff
@@ -0,0 +1,5 @@
+export {Navbar} from "./Navbar/Navbar";
+export {Carousel} from "./carousel/Carousel";
+export {Banner} from "./banner/Banner";
+export {FeaturedCard} from "./featured-card/FeaturedCard";
+export {Footer} from "./footer/Footer";
```

Comment:
Just the way you did here. Just take care image files support default export only I guess